__tests__/app.test.js

```diff
@@ -120,6 +120,42 @@ describe("/api/articles",()=>{
             expect(body.articles).toBeSortedBy("title")
         })
     })
+    test("Get-400 response with bad request if passed in invalid sort_by",()=>{
+
+        return request(app)
+        .get("/api/articles?sort_by=anything&order=asc")
+        .expect(400)
+        .then(({body})=>{
+            expect(body.msg).toBe("Bad request")
+        })
+    })
+
+    test("Get-400 response with bad request if passed in invalid order",()=>{
+
+        return request(app)
+        .get("/api/articles?sort_by=title&order=asec")
+        .expect(400)
+        .then(({body})=>{
+            expect(body.msg).toBe("Bad request")
+        })
+    })
+
+    test("GET-200 response with articles filterd by topic",()=>{
+        return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({body})=>{
+            expect(body.articles.length).toBe(12)
+        })
+    })
```

Comment:
There is another test needed here. What if I request a topic that is valid but has no articles?